main.star

```diff
@@ -127,29 +134,33 @@ def run(plan, args={}):
                 global_tolerations=global_tolerations,
                 persistent=persistent,
                 observability_helper=observability_helper,
-                interop_params=interop_params,
+                supervisors_params=l2_supervisors_params,
                 registry=registry,
             )
         )
 
-    supervisor = None
-    if interop_params.enabled:
-        supervisor = op_supervisor_launcher.launch(
-            plan,
-            l1_config_env_vars,
-            optimism_args.chains,
-            l2s,
-            jwt_file,
-            interop_params.supervisor_params,
-            observability_helper,
+    for superchain_params in optimism_args.superchains:
+        superchain_launcher.launch(
+            plan=plan,
+            params=superchain_params,
+        )
+
+    for supervisor_params in optimism_args.supervisors:
+        op_supervisor_launcher.launch(
+            plan=plan,
+            params=supervisor_params,
+            l1_config_env_vars=l1_config_env_vars,
+            l2s=l2s,
```

Comment:
for symmetry, we should probably pass only the relevant L2s here.
Not urgent though